### DISCUSSION TITLE
Spatial queries possible?

### DISCUSSION THREAD

**Discussion by webwurst** [spaceuptech/space-cloud#1558] [General] [answered]
Is it possible to store and query spatial data like [GeoJSON](https://geojson.org/)? For example with the help of [PostGIS](https://postgis.net/). I would be fine installing that on my own, just wondering if those data types and queries are possible then.

**Comment by YourTechBud** [comment, marked as answer]
Not possible right now. However you can use prepared sql queries (ref - https://docs.spaceuptech.com/storage/database/prepared-queries/) for such use cases and track those tables in schemaless mode (define 0 or 1 columns in SC schema and create the table in the db directly)
